Codigo/SistemaCombate.bas

```diff
@@ -1833,6 +1833,45 @@ Public Function PuedeAtacarNPC(ByVal AttackerIndex As Integer, ByVal NpcIndex As
                         Exit Function
                 End Select
             End If
+            
+            
+            If NpcList(NpcIndex).flags.AttackedBy <> "" Then
```

Comment:
we should move all this test into a new function and call it from Npc.s.UserCanAttackNpc

Reply:
moved